modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -945,7 +972,13 @@ module.exports = {
             input.content = $.html();
           }
 
+          console.log('pre san');
+          console.log(input.content);
+          console.log('/pre san');
           output.content = self.sanitizeHtml(input.content, finalOptions);
+          console.log('post san');
+          console.log(output.content);
+          console.log('/post san');
 
```

Comment:
@boutell sanitizeHtml is stripping out the `style` tag on both the `table` and `col` elements even though (i think) they are configured to allow them through, can i get a second look?